app/Providers/AppServiceProvider.php

```diff
@@ -19,6 +20,6 @@ public function register(): void
      */
     public function boot(): void
     {
-        //
+        JwtClaims::withoutWrapping();
```

Comment:
I wouldn't do this. Use static $wrap = null on the resource

Reply:
Thank you for pointing this out. I was unaware of this possibility. Changed it in latest revision.